incubator/group/handler.go

```diff
@@ -20,89 +21,56 @@ func NewHandler(k Keeper) sdk.Handler {
 			return handleMsgUpdateGroupComment(ctx, k, msg)
 		case MsgUpdateGroupMembers:
 			return handleMsgUpdateGroupMembers(ctx, k, msg)
+
+		// todo: @aaronc is this message type supposed to be handled or for extensions only?
+		//case MsgCreateGroupAccountBase:
+		//case MsgProposeBase:
+
+		case MsgCreateGroupAccountStd:
```

Comment:
Can we create an interface `MsgCreateGroupAccountI`, switch on that instead, and then have `MsgCreateGroupAccountStd` implement it?

Reply:
implemented